.cargo/config.toml

```diff
@@ -1,2 +1,2 @@
 [target.aarch64-unknown-linux-gnu]
-linker = "aarch64-linux-gnu-gcc"
```

Comment:
Another problem, why comment this?

Reply:
I should not commit this code. I will restore this config. My laptop is linux on Mac(M1). I can not install `aarch64-linux-gnu-gcc` to build and test.

Reply:
Got it, copy.